real_time/surface_src/java_src/Alice/src/main/java/com/roboclub/robobuggy/ui/SensorSwitchPanel.java

```diff
@@ -0,0 +1,32 @@
+package com.roboclub.robobuggy.ui;
+
+import javax.swing.JButton;
+
+import com.roboclub.robobuggy.ros.NodeChannel;
+
+public class SensorSwitchPanel extends RoboBuggyGUIContainer{
+	
+	SensorSwitch gps_switch;
+	SensorSwitch vision_switch;
```

Comment:
![Codacy](https://www.codacy.com/versioned/images/favicon.png) Issue found: [Use explicit scoping instead of the default package private level](https://www.codacy.com/app/Mr-Trevor-Decker/RoboBuggy/file/1779953632/issues/source?bid=2938665&fileBranchId=3059924#l10)